LoopKit/LoopPlugin.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  LoopPlugin.swift
+//  LoopKit
+//
+//  Created by Pete Schwamb on 7/24/19.
+//  Copyright © 2019 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+
+public enum LoopPluginBundleKey: String {
+    case pumpManagerDisplayName = "com.loopkit.Loop.PumpManagerDisplayName"
```

Comment:
Is this really `com.loopkit.Loop.*` or `com.loopKit.LoopKit.*`?

Reply:
I think it's actually Loop specific. The plugin protocols, along with CGMManager and PumpManager protocol (and associated UI version) could at some point be moved out into a separate set of frameworks (LoopInterfaces.framework and LoopUIInterfaces.framework, maybe?).  LoopKit is a toolkit for building insulin delivery apps, and defining the interfaces that the very specific Loop app supports is really a separate concern.